docs/motivation.md

```diff
@@ -1,3 +1,117 @@
 # Motivation
 
-## **Under Construction**
+## The problem
+
+Consider a Kubernetes cluster that has an application deployment, such as a
+MySQL database, which uses a PersistentVolume (PV) to store its data.  Such
+applications are also referred to as stateful applications or
+[StatefulSets](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/).
```

Comment:
We need to refer to applications (pods) with stored state, the StatefulSets reference should not imply explicitly or implicitly that we are restricted to only those forms of deployments.